go.mod

```diff
@@ -43,6 +43,7 @@ require (
 require (
 	cloud.google.com/go/compute/metadata v0.2.1 // indirect
 	github.com/Azure/go-ntlmssp v0.0.0-20220621081337-cb9428e4ac1e // indirect
+	github.com/dgraph-io/ristretto v0.1.1 // indirect
```

Comment:
There are 3 separate require blocks.  Is there a distinction between these 3 blocks and why did you place ristretto in this specific one?

Reply:
Good call! I've merged 2 indirect blocks. For `ristretto`, I just forgot to run `go mod tidy`, it's moved to direct block now.